.travis/smoke-test.sh

```diff
@@ -17,3 +18,6 @@ echo "Smoke testing dummy-api"
 echo "Smoke testing browser"
 [[ "$(curl -sS -H 'Host: unstable.libero.pub' "http://localhost:${HTTP_PORT}/blog/post1" --output /dev/null --write-out '%{http_code}')" == "200" ]]
 [[ "$(curl -sS --header 'Host: unstable.libero.pub' "http://localhost:${HTTP_PORT}/favicon.ico" --output /dev/null --write-out '%{http_code}')" == "200" ]]
+
+echo "Smoke testing pattern library"
+[[ "$(curl -sS -H 'Host: unstable--pattern-library.libero.pub' "http://localhost:${HTTP_PORT}" --output /dev/null --write-out '%{http_code}')" == "200" ]]
```

Comment:
`-H` vs `--header`, choose what you prefer  (these lines tend to be long but extracting scripts is non-trivial)

Reply:
Was a copy and paste, but yes I prefer a proper name when not typing.